<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WynZerowe</class>
 <widget class="QDialog" name="WynZerowe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>123</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lab_1">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Funkcja:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Miejsca zerowe:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="lab_funkcja">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_wynZerowe">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_back">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Powr√≥t</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_back</sender>
   <signal>clicked()</signal>
   <receiver>WynZerowe</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fun_backToMain()</slot>
 </slots>
</ui>
